### PR TITLE
ACCUMULO-4179 - Add GitHub badges to README.md

### PR DESCRIPTION
**Pull request by mikewalch**
Did not add Travis CI badge as builds are not completing in time.

### CODE REVIEW

**Comment by ctubbsii** [comment]
The same sha1 is building fine in http://travis-ci.org/ctubbsii/accumulo/, but the one at http://travis-ci.org/apache/accumulo/ can't seem to complete a build without hanging trying to download all the maven dependencies the first time. And, because Travis won't cache anything until a build completes successfully, it starts fresh, re-downloading all the deps each time (It's very annoying).

File: README.md [hunk 1]
```diff
@@ -113,3 +115,9 @@ with Accumulo 1.6.2 and 1.7.0 all API changes will follow [semver 2.0][12]
 [14]: examples/simple/src/main/java/org/apache/accumulo/examples/simple
 [15]: docs/src/main/resources/examples
 [16]: docs/src/main/asciidoc
+[li]: http://img.shields.io/badge/license-ASL-blue.svg
+[ll]: https://github.com/apache/accumulo/blob/master/LICENSE
```
Comment (madrob) [inline README.md]: Could this maybe link to our license on an apache.org host?

Comment (mikewalch) [inline README.md]: Good idea.  Changed in 896625f3d

**Comment by ctubbsii** [comment]
I'm going to merge this in.

**Comment by ctubbsii** [comment]
Test comment.

**Comment by ctubbsii** [comment]
Test comment

**Pull request merged**